crates/cairo-coverage/src/input/statements_map.rs

```diff
@@ -0,0 +1,71 @@
+use crate::input::data_loader::types::{
+    CodeLocation, CoverageAnnotations, LineRange, ProfilerAnnotations,
+};
+use crate::types::{FileLocation, FunctionName};
+use anyhow::{Context, Result};
+use cairo_lang_sierra::debug_info::Annotations;
+use cairo_lang_sierra::program::StatementIdx;
+use serde::de::DeserializeOwned;
+use std::collections::HashMap;
+
+const PROFILER_NAMESPACE: &str = "github.com/software-mansion/cairo-profiler";
+const COVERAGE_NAMESPACE: &str = "github.com/software-mansion/cairo-coverage";
+
+pub type StatementsMap = HashMap<StatementIdx, StatementOrigin>;
```

Comment:
```suggestion
pub type SierraToCairoMap = HashMap<StatementIdx, StatementOrigin>;
```
if you want to create an alias. It is better to use a newtype.